src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjects/Control.ControlAccessibleObjectTests.cs

```diff
@@ -223,19 +223,24 @@ public void ControlAccessibleObject_Description_GetWithAccessibleDescription_Ret
         Assert.Equal(accessibleDescription, accessibleObject.Description);
     }
 
-    public static IEnumerable<object[]> Handle_Set_TestData()
+    public static TheoryData<IntPtr, Control> Handle_Set_TestData()
     {
-        yield return new object[] { IntPtr.Zero };
-        yield return new object[] { (IntPtr)(-1) };
-        yield return new object[] { (IntPtr)1 };
-        yield return new object[] { (IntPtr)250 };
         Control control = new();
-        yield return new object[] { control.Handle, control };
+        var data = new TheoryData<IntPtr, Control>
```

Comment:
I strongly recomment considering using NRTA - since you're passing `null`, the signatures should look as follows:
```suggestion
        var data = new TheoryData<IntPtr, Control?>
```

Reply:
> I strongly recomment considering using NRTA - since you're passing `null`, the signatures should look as follows:

Thanks for your suggestion, I will change this to use NRTA in another PR, because besides this there are many TestData passing null value